gme/gme.exports

```diff
@@ -54,10 +54,10 @@ gme_hes_type
 gme_kss_type
 gme_nsf_type
 gme_nsfe_type
+gme_rar_type
 gme_sap_type
 gme_spc_type
 gme_vgm_type
 gme_vgz_type
 gme_disable_echo
 gme_set_fade_msecs
-gme_rsn_type
```

Comment:
This, along with the other two, were added at the end deliberately because they are new in 0.6.4 -- so it diffs cleaner against 0.6.3

Reply:
I see, I'll put it back at the end then